src/SFA.DAS.FindApprenticeshipTraining.UnitTests/InnerApi/Requests/WhenBuildingTheGetLocationByFullPostcodeRequest.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.FindApprenticeshipTraining.InnerApi.Requests;
+using System.Web;
+
+namespace SFA.DAS.FindApprenticeshipTraining.UnitTests.InnerApi.Requests
+{
+    public class WhenBuildingTheGetLocationByFullPostcodeRequest
+    {
+        [Test, AutoData]
+        public void Then_The_Request_Is_Correctly_Built(string baseUrl, string fullPostcode)
+        {
+            var actual = new GetLocationByFullPostcodeRequest(fullPostcode)
+            {
+                BaseUrl = baseUrl
+            };
+
+            actual.GetUrl.Should().Be($"{baseUrl}api/postcodes?postcode={fullPostcode}");
+        }
+
+        [Test, AutoData]
+        public void Then_The_Request_Is_Correctly_Encoded_For_Special_Characters(string baseUrl, string fullPostcode)
```

Comment:
This test doesn't guarantee that there's a special character in the url. Can you modify `fullPostcode` to include a special character requiring encoding?